TLM/TLM/State/Options.cs

```diff
@@ -38,8 +38,8 @@ public enum PersistTo {
         /// </remarks>
         public static bool Available = false;
 
-        public static bool individualDrivingStyle = true;
-        public static int recklessDrivers = 3;
```

Comment:
These default vals would get overwritten during deserialization (`OptionsManager.LoadData()`) so removed them to clarify they had no effect here in `Options.cs`.